Source/BidMachineAdapterAd.swift

```diff
@@ -28,7 +28,14 @@ class BidMachineAdapterAd: NSObject {
 
     /// The completion for the ongoing show operation.
     var showCompletion: ((Result<PartnerEventDetails, Error>) -> Void)?
-    
+
+    /// Waterfall item price
+    var price: Double? {
+      let setting = self.request.partnerSettings["price"]
```

Comment:
nit: the `self.` here and a few `self.log` below can be take away

Reply:
Removed `self` from line 34 but the compiler said I couldn't remove the ones on lines 40, 41, and 42 because "adapter is a 'let' constant"
I guess it's confused because the init parameters have the same names as the properties on `self`